llama_index/embeddings/clip.py

```diff
@@ -126,6 +126,9 @@ def _get_image_embeddings(self, img_file_paths: List[str]) -> List[Embedding]:
                 results.append(self._model.encode_image(image).tolist()[0])
```

Comment:
very nit: since we are just looping, why don't we just define the single image endpoint `_get_image_embedding` and then rely on the superclass for the multi-image/batch versions?

Reply:
Fixed 👍🏻